CMakeLists.txt

```diff
@@ -99,7 +103,6 @@ set(LIB_SOURCE_FILES
     src/h3lib/include/constants.h
     src/h3lib/include/coordijk.h
     src/h3lib/include/algos.h
-    src/h3lib/include/h3api.h
```

Comment:
I haven't tested this, but will this still cause clang format to auto format h3api.h?

Reply:
Good catch, it wont be auto formatted.

Reply:
Should this line still be removed?

Reply:
It's now autoformatting h3api.h.in

Reply:
There might be a smarter way, the line is removed such that cmake doesn't try to format a non-existent file. The generated h3api.h is still added to the library (using the var `CONFIGURED_API_HEADER`).

Reply:
Ok, thanks!